inc/saiswitch.h

```diff
@@ -140,6 +140,25 @@ typedef enum _sai_hash_algorithm_t
 } sai_hash_algorithm_t;
 
 /**
+ * @brief Attribute data for SAI_SWITCH_ATTR_RESTART_TYPE
+ */
+typedef enum _sai_switch_restart_type_t
+{
+    /** Planned restart only */
+    SAI_RESTART_TYPE_PLANNED = 0,
+
+    /** Unplanned restart only */
+    SAI_RESTART_TYPE_UNPLANNED = 1,
```

Comment:
I really doubt if someone would have 'unplanned only'. 
More practical options would be: none, planned, any (means planned and unplanned).